src/styles/_header.scss

```diff
@@ -37,5 +37,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: $transition;
```

Comment:
specify which properties get the transition, if you do not, `all` will be used and that hits performance